union.py

```diff
@@ -33,7 +33,7 @@ def union(self):
                   f"{sys.argv[3]}: {res_len[2]}, Time taken: {int(time.time() - start_time)}")
             sys.exit(0)
         else:
-            raise Exception("The sys.args parameter not matched", parser.parser())
+            raise Exception("The parameter not matched", parser.parser())
```

Comment:
grammatical issues

Reply:
**Fixed to :**
- `raise Exception("The parameters are not matching", parser.parser())`